pkg/cosign/tlog.go

```diff
@@ -446,7 +448,7 @@ func VerifyTLogEntryOffline(e *models.LogEntryAnon, rekorPubKeys *TrustedRekorPu
 	if !ok {
 		return errors.New("rekor log public key not found for payload")
 	}
-	err = VerifySET(payload, []byte(e.Verification.SignedEntryTimestamp), pubKey.PubKey)
+	err = VerifySET(payload, []byte(e.Verification.SignedEntryTimestamp), pubKey.PubKey.(*ecdsa.PublicKey))
```

Comment:
Should we be enforcing that all Rekor keys are ECDSA keys earlier on, maybe in GetRekorPubs? Otherwise, it fails during verifications. 
L

Reply:
Yeah, would be better to have up the call stack, agreed. I don't think it can be in GetRekorPubs cause people can populate their own CheckOpts values for them. let me see if I can find a better place.

Reply:
Ok, added them right after we check if the keys are nil or there are none, so added check for them right after that which seems like a good place for it.

Reply:
Wondering if we only need to check the `pubKey` retrieved a couple lines earlier. It probably doesn't matter much, but just wondering if someone's TUF root contains our ECDSA keys plus more Rekor keys